store/src/java/com/zimbra/cs/service/mail/SearchResponse.java

```diff
@@ -150,9 +150,9 @@ int size() {
      * @throws ServiceException error
      */
     void add(ZimbraHit zimbraHit) throws ServiceException{
-		add(zimbraHit,false);
+        add(zimbraHit,false);
```

Comment:
revert this if there are no changes in this file

Reply:
this code was not formatted previously, it's formatted now. Should I still change it back?

Reply:
ok